packages/core/src/demo/utils/stackblitz/object-to-string.ts

```diff
@@ -0,0 +1,44 @@
+// Takes a JavaScript object and converts to a string to send to StackBlitz (vs. JSON)
+
+type GenericObject = { [key: string]: any }
+
+export function objectToString(obj: GenericObject | any[] | Date, indent: number = 0): string {
+	let str = indent === 0 ? 'export default ' : ''
+	const padding = ' '.repeat(indent * 2)
+
+	if (Array.isArray(obj)) {
+		str += '[\n'
+
+		obj.forEach((item: any, index: number) => {
+			str += padding + '  ' + objectToString(item, indent + 1)
+			if (index < obj.length - 1) str += ','
+			str += '\n'
+		})
+
+		str += padding + ']'
+	} else if (typeof obj === 'object' && obj !== null) {
+		if (obj instanceof Date) {
+			str += `'${obj.toISOString()}'`
```

Comment:
can we make this shoot out a `new Date(...)` instead of a string?

Reply:
Done